### ISSUE TITLE
Client settings > Custom fields

### ISSUE THREAD

**Issue by chiareu** [boxbilling/boxbilling#230]
We have the ability to add custom fields to customers but we have an issue.
Example: Added VAT field and Company Reg field.
1st - Those custom fields appear on registration form (shall be an option witch fields appear on registration form, default none)
2nd - On the client profile, custom fields are not present at all.

Conclusion. Shall not appear on Registration Form and appear on Client Profile editor.

**Comment by chiareu** [comment]
Now I observed that on Admin interface there are Company details fields by default.
That fields not appear on Client Interface profile event if we change in Admin to Client type: company.
Only way to get that invoicing details is to mail the client.

The problem with custom fields that not appear on Client interface profile persist.

**Comment by 4Giedrius** [comment]
If I understood you correctly you would like that values of custom fields would be displayed in Admin are, when you open client profile?

**Comment by chiareu** [comment]
Let's assume we create a custom field [Bank Account] (on our legislation when we are invoicing companies, that bank account is mandatory)
1. That custom field appear automatically on client register form. We don't want that in this case but in other case with other field name maybe we need that. So a check box to set that will be useful.

2. On the other side, on Client interface / Profile, that custom field are missing. 

3. I set in Admin for that client type: company. Nothing happened? What's the use just for recording?
I expect that on that moment the detailed company fields to appear to client interface to fill the registration number etc.

4. to respond exactly at your question above. I have the first Custom field (Custom_0) configured to Bank Account, and that field are not displayed on Admin interface / Client / Profile.
There are custom fields at the bottom of page but only from 1 to 10. The first (Custom_0) is missing

**Comment by 4Giedrius** [comment]
If you want field to be visible in sign up form just enable it by selecting `Active` checkbox.

1. Custom fields are designed to hold additional information for administrator, so this is the reason why client does not see and can not edit it. 
2. I have allowed `Company VAT` and `Company number` fields to be editable for client: https://github.com/boxbilling/boxbilling/commit/9180cd09ace888dd796ecb318965be54e63a21d3
3. You are right - field `Custom_0` is missing and should not be there. I have fixed that. https://github.com/boxbilling/boxbilling/commit/8dbb38ff73421d10e4e4a8ff27e1eeb02f2103e8
